code/src/components/CompleteList.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { todos } from 'reducers/todos'
+
+import './style/Listed.css'
+
+export const CompleteList = ({ todoIndex }) => {
+  const dispatch = useDispatch()
+  const todo = useSelector((store) => store.todos.list.todos[todoIndex])
+
+  //Deletes item
+  const handleRemoval = (e) => {
+    dispatch(
+      todos.actions.removeItem({
+        todoIndex,
+      })
+    )
+  }
+
+  //Toggles checkbox 
+  const handleChecked = (e) => {
+    dispatch(
+      todos.actions.setDone({
+        todoIndex: todoIndex,
```

Comment:
again here you can shorten this since key and value are the same.
Also I would suggest using the id instead of the index for things like this. Just in case you're modifying the array, then the indexes will change. The id should be unique for specific todos all the time.